setup-files/gerald-push.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 0
-      - run: git fetch
+          ref: '${{ github.ref }}'
+          fetch-depth: '$(node -e "console.log(${{ github.event.commits.length }}.length + 1")'
```

Comment:
Woah, this is very hacky! Haha - is it possible to leave a comment here explaining what's happening so that future folks know how to maintain it?